ql/src/java/org/apache/hadoop/hive/ql/Driver.java

```diff
@@ -155,9 +157,11 @@ public int compile(String command, boolean resetTaskIds) {
     }
   }
 
-  // deferClose indicates if the close/destroy should be deferred when the process has been
-  // interrupted, it should be set to true if the compile is called within another method like
-  // runInternal, which defers the close to the called in that method.
+  /**
+   * @deferClose indicates if the close/destroy should be deferred when the process has been interrupted, it should be
```

Comment:
Maybe a correct javadoc comment? The meaning and the usage of resetTaskIds is not trivial either